charts/avatar-creation/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.3 #0.1.2
+version: 0.1.4 #0.1.3 #0.1.2
```

Comment:
Can you remove the commented out versions?

Reply:
removed